src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -3,6 +3,8 @@
 @using APIViewWeb.Helpers
 @using APIViewWeb.LeanModels;
 @using APIViewWeb.Models
+@using System
+@using System.Text.RegularExpressions

```

Comment:
nit: it doesn't look like this is being used. if that is the case, we can delete it